src/microbe_stage/editor/BarHelper.cs

```diff
@@ -0,0 +1,72 @@
+using Godot;
+
+/// <summary>
+///   Used to access the color and icon of a bar from a provided dictionary
+/// </summary>
+public static class BarHelper
+{
+    public static Color GetBarColour(string type, string name, bool production)
+    {
+        switch (type)
+        {
+            case "ATP":
```

Comment:
I think that the type should be made an enum so that it is known what the valid values are.